### ISSUE TITLE
Extend the weekly report to include epics status and work progression

### ISSUE THREAD

**Issue by cmoulliard** [snowdrop/team#303]
## Feature request

Extend the team weekly report to include epics status and work progression.
So, include a new table reporting the following information: 
  - column 1  : Epic title + link
  - column 2 : Domain. Where domain is one of the following values: dekorate, spring on quarkus, devex, spring cloud, ...
  - column 3 : Owner
  - column 4 : Delivery/release date
  - column 5 : %. It represents the progression of the work/tasks. It could be calculated for the moment as the sum of the tasks accomplished/number of tasks. If all the tasks are accomplished then the% will be : 100, No tasks: 0, ....
  - column 6: Releasing quarter: Q1-2021, Q2-2020, ...

**Comment by gytis** [comment]
This sounds like a duplicate of a Zenhub roadmap.

**Comment by cmoulliard** [comment]
Dont think so. Make then a try

**Comment by gytis** [comment]
Zenhub roadmap has projects and epics, their deadline and the swim-lanes with their progress. It's useless now because nobody uses epics. But the new report would have the same problem.

**Comment by cmoulliard** [comment]
This is why as requested 2 weeks ago in my email, team should use now epics to group/organize their tasks

**Comment by gytis** [comment]
Then the Zenhub report should be usable. I thought we started to used Zenhub specifically for its GitHub integrations and reports?

**Comment by cmoulliard** [comment]
Make then a test to prove that you can generate the requested report

**Comment by gytis** [comment]
Here is the epics roadmap https://app.zenhub.com/workspaces/snowdrop-5e13555db24025e26165460f/roadmap. People have to include their epics there. See "Implement Spring Data REST Extension" for example.

Here is a last week's issues report: https://app.zenhub.com/workspaces/snowdrop-5e13555db24025e26165460f/reports/control?df=11-01-2020&dr=custom&dt=11-07-2020&ep=Z2lkOi8vcmFwdG9yL1BpcGVsaW5lLzA&prs=&r=&s=day&sp=Z2lkOi8vcmFwdG9yL1BpcGVsaW5lLzIwNzA5NzU

**Comment by cmoulliard** [comment]
> Here is the epics roadmap https://app.zenhub.com/workspaces/snowdrop-5e13555db24025e26165460f/roadmap. People have to include their epics there. See "Implement Spring Data REST Extension" for example.

We dont really see then the progression. Do I miss something. Can you create an epic with 4 tasks where 2 have been accomplished to see the result ?

**Comment by cmoulliard** [comment]
> Here is a last week's issues report: https://app.zenhub.com/workspaces/snowdrop-5e13555db24025e26165460f/reports/control?df=11-01-2020&dr=custom&dt=11-07-2020&ep=Z2lkOi8vcmFwdG9yL1BpcGVsaW5lLzA&prs=&r=&s=day&sp=Z2lkOi8vcmFwdG9yL1BpcGVsaW5lLzIwNzA5NzU

Results are not aggregated/user so I cannot measure what you or ozzy or ... associates have accomplished

**Comment by gytis** [comment]
For that we already have a weekly report with every issue. The way I understand your request is that you want to see how the epics are progressing.

**Comment by cmoulliard** [comment]
> to see how the epics are progressing.

Yes. I would like to have an aggregated view of the epics grouped by owner and where we can see by example

 Epic1 | Spring on Quarkus | Gytis | 30-Nov | 100% | Q1-2021
 Epic2 | DevExp | Chris | - | 50% | Q1-2021
 Epic2 | Prod | Antonio | - | 50% | Q1-2021
 ...

OR

Gytis
 Epic1 | Spring on Quarkus | 25-Nov | 100% | Q1-2021
 Epic5 | DevExp | - | 50% | Q1-2021
 Epic8 | Prod | - | 50% | Q1-2021

Chris
 Epic15| Quarkus | 12-Nov | 100% | Q1-2021
 Epic22 | DevExp | - | 0% | Q2-2021
 Epic66 | Prod | - | 0% | Q1-2021

**Comment by cmoulliard** [comment]
We should review/discuss this epic next week and take a decision @gytis
